subsys/net/l2/wifi/Kconfig

```diff
@@ -119,3 +119,16 @@ config WIFI_MGMT_AP_MAX_NUM_STA
 	help
 	  Maximum number of stations allowed in station table. New stations will be
 	  rejected after the station table is full.
+
+config WIFI_MGMT_BSS_MAX_IDLE_TIME
```

Comment:
We already have this in `modules/hostap/Kconfig`, we should keep only one owner.